### ISSUE TITLE
Can't 'stack all items' in the field

### ISSUE THREAD

**Issue by esotericist** [prasoc/BetterUI#29]
Slight issue of 'too many bindings, not enough buttons' I imagine, but I have had the issue of a couple of stacks of an item I wanted to combine, but I needed to either disable the BUI inventory module or head to a bank to fix it.

Don't know how we can address it. #28 might let us reclaim that binding when the Mark As Junk functionality is disabled (and that setting toggle doesn't seem to require a /reloadui, although I don't know if that will still be the case when #28 is resolved)

**Comment by trentapple** [comment]
Yeah, it is a tough one -- I think we need to at least have it show stack all items as the keybind if junk is disabled. Alternatively we can add it to the actions "Y" menu.
